packages/mini-cart/src/mini-cart.tsx

```diff
@@ -156,18 +163,21 @@ export function MiniCart( {
 					removeProductFromCart={ handleRemoveProduct }
 					responseCart={ responseCart }
 				/>
-				<MiniCartTotal responseCart={ responseCart } />
+				{ shouldRenderEmptyCart && <EmptyCart /> }
```

Comment:
Re: my comment above, if you change the type of `emptyCart` to `React.ReactNode` (or similar), then this could be a lot simpler. Is there a reason to make this an element type instead?

```suggestion
				{ shouldRenderEmptyCart && emptyCart }
```

Reply:
Yes, thank you @sirbrillig. Just added a refactor [commit below](https://github.com/Automattic/wp-calypso/pull/70588/commits/36640d949d18d7873db89c3af1709b0296abf6f8) and [small followup](https://github.com/Automattic/wp-calypso/pull/70588/commits/36640d949d18d7873db89c3af1709b0296abf6f8)